src/contexts/bridge.context.tsx

```diff
@@ -0,0 +1,118 @@
+import { BigNumber, ContractTransaction, PayableOverrides } from "ethers";
+import { createContext, FC, useContext, useEffect, useState } from "react";
+
+import { useEnvContext } from "src/contexts/env.context";
+import { useProvidersContext } from "src/contexts/providers.context";
+import { Bridge, Bridge__factory } from "src/types/contracts/bridge";
+
+interface BridgeContext {
+  bridge: (
+    token: string,
+    amount: BigNumber,
+    destinationNetwork: number,
+    destinationAddress: string,
+    overrides?: PayableOverrides & { from?: string | Promise<string> }
```

Comment:
I wouldn't expose this `overrides` param as we are not going to use it.

Reply:
They are being used [as you can see here](https://github.com/hermeznetwork/internal-tools/blob/main/FE-tools/generateClaimBridge.js#L58). And more important, if you miss them the contract returns an error.

Reply:
Right, thanks for pointing that! The things it that we only need to send this `{ value: amount }` when we are sending `ether`. If we are sending an ERC-20 token this is not needed. As I proposed in Slack, I'll keep this simple because we are missing the ERC-20 approval as well, so I'll say that we only support `ETH` bridges in this iterarion:

```ts
if (tokenAddress !== ethers.constants.AddressZero) { {
  throw new Error("ERC-20 tokens are not supported yet")
}

return bridgeContract.bridge(tokenAddress, amount, destinationNetwork, destinationAddress, { value: amount });
```

I'm creating another issues to support ERC-20 tokens on other iteration.

Reply:
Ok, that makes sense. Done!